target/linux/ramips/image/mt7621.mk

```diff
@@ -332,6 +346,24 @@ define Device/iodata_wn-ax1167gr
 endef
 TARGET_DEVICES += iodata_wn-ax1167gr
 
+define Device/iodata_wn-dx1167r
+  MTK_SOC := mt7621
```

Comment:
Rename to "SOC" due to 58e71e322bc0e0edd08157075e9022c15ae33ec5.

Reply:
done